Update Readme.md (#1)
DevExpress-Examples/out-of-maintenance-wpf-dock-layout-manager-support-mvvm-and-implement-a-custom-style-for-document-pa: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E3922)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
# WPF Dock Layout Manager - Support MVVM and Implement a Custom Style for DocumentPanel (Google Chrome-like "New Tab" Button)

> This sample uses MVVM and provide your with own custom style for a [DocumentPanel](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DocumentPanel). A [DocumentPanel](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DocumentPanel) header includes a "new tab" button like Google Chrome.

Reviewer: ```suggestion
This sample uses MVVM and implements a custom style for [DocumentPanel](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DocumentPanel). A [DocumentPanel](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DocumentPanel) header includes a "new tab" button similar to Google Chrome.
```
